bridge/package.json

```diff
@@ -105,6 +109,7 @@
     "minimist": "1.2.5",
     "npm-run-all": "4.1.5",
     "prettier": "2.5.1",
+    "run-script-os": "^1.1.6",
```

Comment:
Please remove this dependency, I think we can handle it without any new dependency